woocommerce/woocommerce-gutenberg-products-block
# Shopper → Cart → Can update product quantity

Fixes #5755

### Notes

This PR adds an e2e test to verify that shoppers can update product quantity.

### Testing

#### Automated Tests

### Manual Testing

1. Check out this PR.
2. Start Docker Desktop.
3. Run `npm run wp-env destroy && npm run wp-env start` to start a fresh e2e environment.
4. Run `npm run test:e2e -- tests/e2e/specs/shopper/cart-update-product-quantity.test.js` to run only this test case.
5. Run `npm run test:e2e` to run the whole test suite.

### TODO

- [X] Find why test passes only in interactive mode, otherwise, we have the following error: `TimeoutError: Element a.wc-block-cart__submit-button not found`
- [X] Add qty using the "add" button
- [X] decrease qty using the "decrease" button
